PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -18,6 +26,18 @@ Router is only certified to work with EPrints 3.3.
 EPrints is configured by default to accept incoming requests via SWORDv2, so the router's deposit mechanism
 will automatically work.  
 
+### Eprints RIOXX Plugin
```

Comment:
You need to modify the preceding sentence to make clear that there are 2 options for Eprints - the native option (which is already configured) and the RIOXX option which gives additional benefits.